oauthlib/oauth2/rfc8628/endpoints/device_authorization.py

```diff
@@ -32,14 +34,7 @@ class DeviceAuthorizationEndpoint(BaseEndpoint):
     themselves.
     """
 
-    def __init__(
-        self,
-        request_validator,
-        verification_uri,
-        expires_in=1800,
-        interval=None,
-        verification_uri_complete=None,
-    ):
+    def __init__(self, request_validator, verification_uri, expires_in=1800, interval=None, verification_uri_complete=None, user_code_generator: Callable[[None], str] = None):
```

Comment:
I think the previous hanging style format looks better and seems more readable.

Reply:
I agree but this is what ruff formatted it to.

I'll play around with the ruff config

Reply:
@auvipy I locally changed the max line length to 99 for ruff. The 255 is too high. Would need a seperate pr to format the entire repo

[commit here](https://github.com/oauthlib/oauthlib/pull/885/commits/6862c41f3994f770c3a1fbdb399a689f6e98a520)

Reply:
@auvipy 
Ideally the ruff config should have a lower max line length but running it on the code base leads to too many violations in this repo so just changed the files related to this pr in regards to formatting

[commit here](https://github.com/oauthlib/oauthlib/pull/885/commits/51ddcf652642c29d4b70fc07c7258d5f08509101)